data/images/particles/wind.sprite

```diff
@@ -0,0 +1,26 @@
+(supertux-sprite

```

Comment:
Can you please check the indentation of this file and fix it (because if we merge it now, this won't happen)

Oh, and add `[ci skip]` after your commit message so that we don't waste unnecessary integration cycles.